.changeset/new-news-count.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lit-labs/ssr': patch
+---
+
+Make attribute handling case-insensitive as it is in the browser.
```

Comment:
Nit: Worth updating this changeset to similarly specify "for HTML elements" or something to differentiate from the SVG namespace.

Reply:
Good suggestion!